invokeai/frontend/web/src/features/canvas/hooks/useCanvasHotkeys.ts

```diff
@@ -92,39 +91,35 @@ const useInpaintingCanvasHotkeys = () => {
     [activeTabName, shouldShowBoundingBox]
   );
 
-  const onKeyDown = useCallback((e: KeyboardEvent) => {
-    if (
-      e.repeat ||
-      e.key !== ' ' ||
-      isInputElement(e.target as HTMLElement) ||
-      !isElChildOfCanvasTab(e.target as HTMLElement)
```

Comment:
Removing this condition means the space hotkey now interferes with accessibility - you cannot use space to interact with focused UI elements (like buttons).